frontend/scripts/replace-env-at-runtime.sh

```diff
@@ -1,20 +1,45 @@
-#!/bin/bash
+#!/bin/sh
 
-# check first argument, if it is an existing file
-if [ -r $1 ]
+usage=`cat <<-EOF
+Replace the /*@preserve __ENV_INJECT_MARK__*/ marker in an conquery index.html with environment variables.
+If an optional .env-file is supplied it will be read before subsitution. 
+
+Usage:
+$0 <index.html> [.env-file]
+EOF
+`
+
+# Check for one or two arguments or exit 
+if  [ \( "$#" -lt 1 \) -o \( "$#" -gt 2 \) ] 
+then
+    echo "$usage"
+    exit 1
+fi
+
+# Check if env file is provided and readable
+if [ "$#" -eq 2 ]
```

Comment:
Why isn't the env already sourced when invoking this file?

Reply:
You might also want to just use `env` to invoke it

Reply:
but then, i would need to run the following command in a subshell

Reply:
wobei man bei `env` nicht direkt ein env-File angeben kann

Reply:
`env $(cat ./env) replace-env-at-runtime.sh`?

Reply:
Habs probiert, Dafür muss man die env-datei vorher säubern (Kommentare, vmtl auch zeilen umbrüche)

Reply:
So würde es gehen: `(source .env.e2e ; ./scripts/replace-env-at-runtime.sh build/index.html.test)`

Reply:
Fixt du das noch?

Reply:
Ich hatte es auf dem Server nochmal getestet und muss das sourcen des env-files mit `set -+a` umranden